### PR TITLE
Add program files

### PR DESCRIPTION
**Pull request by alukash**
Create program to serialize collections to xml files

### CODE REVIEW

File: SerializeXML/Program.cs [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+			var xmlOverrides = new XmlAttributeOverrides();
+			var xmlAttribs = new XmlAttributes();
+			xmlAttribs.XmlType = new XmlTypeAttribute("attr");
+			xmlOverrides.Add(typeof(Transmission.Types), xmlAttribs);
+			XmlSerializer xmlSerializer = new XmlSerializer(typeof(List<Vehicle>), xmlOverrides);
+			FileStream fs = new FileStream(file, FileMode.OpenOrCreate);
+			xmlSerializer.Serialize(fs, vehicles);
+			fs.Close();
+		}
+		private static void CreateXml(string file, List<Vehicle> vehicles)
```
Comment (ViktarDrummer) [inline SerializeXML/Program.cs]: please add extra space.

File: SerializeXML/Program.cs [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+				vehicleElem.Add(powerElem);
+				vehicleElem.Add(volumeElem);
+				vehiclesElem.Add(vehicleElem);
+			});
+			xdoc.Add(vehiclesElem);
+			xdoc.Save(fs);
+			fs.Close();
+		}
+
+		private static void CreateXml2(string file, List<Vehicle> vehicles)
```
Comment (ViktarDrummer) [inline SerializeXML/Program.cs]: Just thoughts: we are breaking single responsibility here a little bit. 

Those methods looks reusable and probably better to put them into different class, or classes. For this task it's not critical.

**Review by ViktarDrummer — APPROVED** [review]

File: SerializeXML/Program.cs (hunk 1 above)
Comment (alukash) [reply SerializeXML/Program.cs]: Done

File: SerializeXML/Program.cs (hunk 2 above)
Comment (alukash) [reply SerializeXML/Program.cs]: Created new "Vehicles" class.
Moved the methods to the new class.

**Pull request merged**